### ISSUE TITLE
version not found: read-json@0.1.0

### ISSUE THREAD

**Issue by bmatusiak** [twbs/bootstrap#19596]
i have a issue following the instructions on the getting started page.

`version not found: read-json@0.1.0`

i tried to search for the issue, but i have not see any post on this

```
bmatusiak:~/workspace $ git clone https://github.com/twbs/bootstrap.git
Cloning into 'bootstrap'...
remote: Counting objects: 93466, done.
remote: Compressing objects: 100% (64/64), done.
remote: Total 93466 (delta 28), reused 0 (delta 0), pack-reused 93402
Receiving objects: 100% (93466/93466), 91.62 MiB | 19.86 MiB/s, done.
Resolving deltas: 100% (60181/60181), done.
Checking connectivity... done.
bmatusiak:~/workspace $ npm install -g grunt-cli
/home/ubuntu/.nvm/versions/node/v4.4.0/bin/grunt -> /home/ubuntu/.nvm/versions/node/v4.4.0/lib/node_modules/grunt-cli/bin/grunt
grunt-cli@1.1.0 /home/ubuntu/.nvm/versions/node/v4.4.0/lib/node_modules/grunt-cli
├── nopt@3.0.6 (abbrev@1.0.7)
├── resolve@1.1.7
└── findup-sync@0.3.0 (glob@5.0.15)
bmatusiak:~/workspace $ cd bootstrap/
bmatusiak:~/workspace/bootstrap (master) $ npm install
npm WARN deprecated lodash@0.9.2: Grunt needs your help! See https://github.com/gruntjs/grunt/issues/1403.
npm WARN deprecated npmconf@2.1.1: this package has been reintegrated into npm and is now out of date with respect to npm
npm WARN deprecated graceful-fs@1.2.3: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
 
> phantomjs@1.9.19 install /home/ubuntu/workspace/bootstrap/node_modules/grunt-contrib-qunit/node_modules/grunt-lib-phantomjs/node_modules/phantomjs
> node install.js

PhantomJS not found on PATH
Downloading https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-linux-x86_64.tar.bz2
Saving to /tmp/phantomjs/phantomjs-1.9.8-linux-x86_64.tar.bz2
Receiving...
| [================------------------------] 40% 0.0s/
Received 12854K total.
Extracting tar contents (via spawned process)
Removing /home/ubuntu/workspace/bootstrap/node_modules/grunt-contrib-qunit/node_modules/grunt-lib-phantomjs/node_modules/phantomjs/lib/phantom
Copying extracted folder /tmp/phantomjs/phantomjs-1.9.8-linux-x86_64.tar.bz2-extract-1458748219665/phantomjs-1.9.8-linux-x86_64 -> /home/ubuntu/workspace/bootstrap/node_modules/grunt-contrib-qunit/node_modules/grunt-lib-phantomjs/node_modules/phantomjs/lib/phantom
Writing location.js file
Done. Phantomjs binary available at /home/ubuntu/workspace/bootstrap/node_modules/grunt-contrib-qunit/node_modules/grunt-lib-phantomjs/node_modules/phantomjs/lib/phantom/bin/phantomjs
npm ERR! Linux 4.2.0-c9
npm ERR! argv "/home/ubuntu/.nvm/versions/node/v4.4.0/bin/node" "/home/ubuntu/.nvm/versions/node/v4.4.0/bin/npm" "install"
npm ERR! node v4.4.0
npm ERR! npm  v2.14.20

npm ERR! version not found: read-json@0.1.0
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/workspace/bootstrap/npm-debug.log
bmatusiak:~/workspace/bootstrap (master) $ 
```

**Comment by bmatusiak** [comment]
https://github.com/twbs/bootstrap/issues/19595

**Comment by arechsteiner** [comment]
Caused by https://github.com/uber/npm-shrinkwrap/issues/103

Nothing wrong with your setup. I've spent some time troubleshooting, installing and reinstalling node and npm and finally figured out which package is responsible. I guess this will resolve itself in a short while (hopefully)! I assume this breaks lots of builds all over the world.

**Comment by Johann-S** [comment]
It's because of this : https://medium.com/@azerbike/i-ve-just-liberated-my-modules-9045c06be67c#.cf8jrwiz3

**Comment by bmatusiak** [comment]
ok so this worked for me...

before `npm install`

do `npm install https://github.com/uber/npm-shrinkwrap.git`

and should temp fix this issue

**Comment by cvrebert** [comment]
Fixing #18559 will sidestep this clusterfsck.

**Comment by cvrebert** [comment]
Fixed by #19604 & #19605.
